tests/jobsautoreport/test_report.py

```diff
@@ -219,11 +219,11 @@
     number_of_e2e_or_subsystem_periodic_jobs=3,
     number_of_successful_e2e_or_subsystem_periodic_jobs=2,
     number_of_failing_e2e_or_subsystem_periodic_jobs=1,
-    success_rate_for_e2e_or_subsystem_periodic_jobs=66.67,
+    success_rate_for_e2e_or_subsystem_periodic_jobs=66.66666666666667,
```

Comment:
I never used it, pytest provides `approx` to test floats, it would make the tests more reliable:
https://docs.pytest.org/en/latest/reference/reference.html#pytest-approx

Reply:
I believe this is problematic because we compare Report against Report, and `approx` doesn't know how to handle custom objects @adriengentil

Reply:
have you looked at https://github.com/pytest-dev/pytest/issues/6632#issuecomment-580507745 ?

Reply:
Nice

Reply:
I don't see ho to get it to work in a nice way, this recursive approach doesn't work for lists, tuples.. it created a mess. how about rounding the numbers in the test?